README.md

```diff
@@ -38,7 +38,7 @@ Please refer to the [installation guide](https://chainermn.readthedocs.io/en/lat
 You can invoke MNIST example with four workers by the following command:
 
 ```
-mpiexec -n 4 python examples/mnist/train_mnist.py
+mpiexec -n 4 python examples/mnist/train_mnist_data_parallel.py
```

Comment:
I'm afraid we should not change this line.
Data parallel model is an advanced feature and users should first see the basic example of MNIST.